Bottom: 035ab71dcd54559f84998482376cbc72e0a96674
Top:    16ab6cde6d4ccead7732b4cb1cd959c49c30d9cd
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-10 04:01:33 +0000

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!










---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index 59a9a29..7843ca4 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -332,13 +332,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
